crates/client/src/rr/dnssec/keypair.rs

```diff
@@ -407,10 +407,9 @@ impl<K: HasPrivate> KeyPair<K> {
                                 }
                                 part = &part[1..];
                             }
-                            for _ in 0..(part_len - part.len()) {
-                                // Pad with zeros. All numbers are big-endian here.
-                                ret.push(0x00);
-                            }
+
+                            // Pad with zeros. All numbers are big-endian here.
+                            ret.extend((0..(part_len - part.len())).into_iter().map(|_| 0x00));
```

Comment:
Should this also get the iter::repeat() treatment?

Reply:
yea, I can do that. I can do it in about an hour.